.prettierrc.js

```diff
@@ -0,0 +1,9 @@
+/** @type {import('prettier').Config} */
+const config = {
+  semi: false,
+  singleQuote: true,
+  trailingComma: 'all',
+  plugins: [require.resolve('prettier-plugin-organize-imports')],
```

Comment:
Normally `prettier` plugins are automatically detected by `prettier` we're installed, but we're using [`pnpm`](https://pnpm.io/) as our package manager, so we need to manually `require.resolve` the import here.

See https://github.com/prettier/prettier/issues/8056 for more information.